### PR TITLE
Rundler v0.3 Feature Branch

### PR DESCRIPTION
**Pull request by dancoombs**

### CODE REVIEW

File: bin/rundler/src/cli/builder.rs [hunk 1]
```diff
@@ -115,14 +130,48 @@ pub struct BuilderArgs {
...
+    /// Only used when BUILDER_SENDER is "raw"
     #[arg(
         long = "builder.submit_url",
         name = "builder.submit_url",
         env = "BUILDER_SUBMIT_URL"
     )]
     pub submit_url: Option<String>,
 
+    /// If present, the url of the ETH provider that will be used to check
+    /// transaction status. Else will use the node http for status.
```
Comment (dancoombs) [inline bin/rundler/src/cli/builder.rs:142]: Fix

File: bin/rundler/src/cli/builder.rs [hunk 2]
```diff
@@ -57,13 +57,28 @@ pub struct BuilderArgs {
...
     #[arg(
         long = "builder.private_key",
         name = "builder.private_key",
         env = "BUILDER_PRIVATE_KEY"
     )]
     private_key: Option<String>,
 
+    /// Private keys to use for signing transactions
+    ///
+    /// Cannot use both `builder.private_key` and `builder.aws_kms_key_ids` at the same time.
```
Comment (dancoombs) [inline bin/rundler/src/cli/builder.rs:73]: Fix

File: docs/cli.md [hunk 3]
```diff
@@ -216,11 +227,11 @@ Here are some example commands to use the CLI:
...
-$ ./rundler node --entry_points 0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789 --chain_id 1337 --max_verification_gas 10000000
+$ ./rundler node --chain_id 1337 --max_verification_gas 10000000 --disable_entry_point_v0_6
 
 # Run the RPC subcommand with custom options and enable JSON logging. The builder and pool will need to be running before this starts.
-$ ./rundler rpc --node_http http://localhost:8545 --log.json
+$ ./rundler rpc --node_http http://localhost:8545 --log.json --disable_entry_point_v0_6
 
 # Run the Pool subcommand with custom options and specify a mempool config file
-$ ./rundler pool --max_simulate_handle_ops_gas 15000000  --mempool_config_path mempool.json --node_http http://localhost:8545 --chain_id 8453
+$ ./rundler pool --max_simulate_handle_ops_gas 15000000  --mempool_config_path mempool.json --node_http http://localhost:8545 --chain_id 8453 --disable_entry_point_v0_6
```
Comment (dancoombs) [inline docs/cli.md:236]: Check if these work

**Review by alex-miao — APPROVED** [review]

**Review by 0xfourzerofour — APPROVED** [review]
Good work on this sir

**Pull request merged**